server/order/router/order_router.js

```diff
@@ -8,8 +8,12 @@ const orderController = require('../controller/order_controller');
 const router = express.Router();
 
 router.post('/', (req, res) => {
```

Comment:
Please don't use req, res. We have to use full name instead of these.
Like request, response.